plugins/de.ovgu.featureide.fm.core/.classpath

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
+	<classpathentry exported="true" kind="lib" path="lib/uvl-parser.jar"/>
```

Comment:
License of that library?

Reply:
It is licensed under the MIT license, so it should be compatible with the LGPL. Where are the licenses of the used libraries documented? I haven't found the place.

Reply:
I added a license file next to the library. Still someone should copy it into the features, so people can see it during installation.